Automattic/wp-calypso
# Make focus state clearer on some controls

#### Changes proposed in this Pull Request

* Updated CSS on `AppsBadge` and `Task` ("Hide this" button) components so that their focus state is clearer. 

#### Testing instructions



"Hide this" button:
* Head to My Home.
* Click on a card and press tab until the "Hide this" button has focus.
* When the "Hide this" button receives focus, it should have a blue solid outline.
![Screen Shot 2022-02-15 at 5 03 36 PM](https://user-images.githubusercontent.com/797888/154028557-ed0d7517-996f-43cd-a0c9-459937f55d9b.png)


`AppsBadge`:
* Head to My Home.
* On the right sidebar, beneath Quick Links, find the section Wordpress apps.
* Click on the title "Wordpress apps" and press tab until the `AppsBadge` component receives focus.
* When the `AppsBadge` component receives focus, it should have the browser's default outline (dotted black outline).
![Screen Shot 2022-02-15 at 5 17 56 PM](https://user-images.githubusercontent.com/797888/154031086-48339e1e-ce75-4ae4-8499-0db725119159.png)





Related to #53617
